### ISSUE TITLE
deck: `getLineDashArray` has been removed. Use `PathStyleExtension` instead.

### ISSUE THREAD

**Issue by R-D-P** [uber/nebula.gl#312]
#### When using the editableGeojsonLayer from nebula.gl with Deck.gl version 8, the following error appears 
" deck: `getLineDashArray` has been removed. Use `PathStyleExtension` instead "

**Comment by R-D-P** [comment]
` const editableLayer = new EditableGeoJsonLayer({
      id: 'geojson-layer-editable',
      selectedFeatureIndexes: selectedFeatureIndexes || [],
      data: geojson,
      mode: mode,
      extensions: [new PathStyleExtension({ dash: true })],
      getDashArray: [3, 2],
      dashJustified: true,
      onEdit: ({ updatedData, editType, editContext }) => {
        if (this.props.page === 'datasetMap') {
          this.addMultiplePolygon(updatedData, editType, editContext)
        } else {
          this.addSinglePolygon(updatedData, editType, editContext)
        }
      },
    })`

I event tried using the extensions as described in the docs of deck.gl, however still get the error.
Any help would be appreciated.

**Comment by georgios-uber** [comment]
Probably the property is not being forwarded.

**Comment by georgios-uber** [comment]
@supersonicclay can we find a more generic way to forward properties? Maybe forward everything that we don't know about so we are fwd compatible.

**Comment by supersonicclay** [comment]
You can accomplish this by using `_subLayerProps` from deck.gl:
https://deck.gl/#/documentation/deckgl-api-reference/layers/composite-layer?section=_sublayerprops-object-experimental to override `EditableGeoJson`'s sub layer properties.

Something like this:
```
new EditableGeoJsonLayer({
  _subLayerProps: {
    geojson: {
      // any prop supported by deck.gl's GeoJsonLayer including:
      extensions: {...}
    }
  }
});
```

We encourage folks to start using `_subLayerProps` now since they give you the ability to pass props directly to the deck.gl layers that nebula uses under the covers.

The following sublayers are rendered by `EditableGeoJsonLayer`:
* `geojson`: a GeoJsonLayer that renders the geometry passed in the `data` prop)
* `guides`: a GeoJsonLayer that renders the edit handles and tentative geometries being edited)
* `tooltips`: a TextLayer for tooltips (e.g. the distance tooltip for the new measure distance mode)

**Comment by supersonicclay** [comment]
Also, we just released `0.17.0` with a couple of other fixes for deck.gl 8
